cli/js/fetch.ts

```diff
@@ -266,48 +80,73 @@ export class Response implements domTypes.Response {
     headersList: Array<[string, string]>,
     rid: number,
     redirected_: boolean,
-    body_: null | Body = null
+    readableStream_: domTypes.ReadableStream | null = null
   ) {
     this.trailer = createResolvable();
     this.headers = new Headers(headersList);
     const contentType = this.headers.get("content-type") || "";
 
-    if (body_ == null) {
-      this.body = new Body(rid, contentType);
+    if (readableStream_ == null) {
```

Comment:
there's an implicit capture of our `rid` here.
we pump the read date from that rid into the readable stream that we use to create the body